### PR TITLE
Chore/global eslint

### PR DESCRIPTION
**Pull request by literat**
Lint all project javascript files.

### CODE REVIEW

File: examples/web/postcss.config.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+// eslint-disable-next-line import/no-unresolved
```
Comment (literat) [inline examples/web/postcss.config.js:1]: ESLint cannot resolve this dependency because example has its own package.json.

Comment (adamkudrna) [reply examples/web/postcss.config.js:1]: Would you please drop it as a comment right into the code?

Comment (literat) [reply examples/web/postcss.config.js:2]: Ok, added.

File: packages/web-react/.eslintrc.js [hunk 2]
```diff
@@ -28,16 +27,14 @@ module.exports = {
   },
 
   extends: [
-    '@lmc-eu/eslint-config-base',
-    '@lmc-eu/eslint-config-base/optional',
-    'prettier',
+    '../../.eslintrc.js',
     'plugin:react/recommended', // eslint react rules (github.com/yannickcr/eslint-plugin-react)
     'plugin:jsx-a11y/recommended', // accessibility plugin
     // Prettier plugin and recommended rules
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/web-react/.eslintrc.js]: Yes, fixed.

**Pull request merged**